Makefile

```diff
@@ -102,7 +102,7 @@ build: get $(BINDIR)/$(BINNAME) ## Build go code
 	@printf "\033[32m-------------------------------------\n BUILD SUCCESS\n-------------------------------------\033[0m\n"
 
 $(BINDIR)/$(BINNAME): $(SRC) $(ASSETS) $(WEB_ASSETS)
-	CGO_ENABLED=0 go build -ldflags '$(LDFLAGS)' -o $(BINDIR)/$(BINNAME) .
+	CGO_ENABLED=1 go build -ldflags '$(LDFLAGS)' -o $(BINDIR)/$(BINNAME) .
```

Comment:
Can we keep `CGO_ENABLED=0` ?

Reply:
I couldn't build WASMEdge with `CGO_ENABLED=1`: 

```
github.com/second-state/WasmEdge-go/wasmedge: build constraints exclude all Go files in /Users/<redacted>/go/pkg/mod/github.com/second-state/!wasm!edge-go@v0.13.0/wasmedge```

Reply:
ok wasmedge-go seems to be a binding on the C++ wasmedge project